justfile

```
@@ -7,10 +7,21 @@ export DEBUG := env_var_or_default('DEBUG', "")
 lint:
   shellcheck -x */*.sh jobrunner/bashrc
 
-build:
-  make test-image keys/testuser
+build: testuser-key build-test-image
 
-test:
+build-test-image:
+  #!/usr/bin/env bash
+  test packages.txt -ot .test-image -a core-packages.txt -ot .test-image -a purge-packages.txt -ot .test-image -a build-lxd-image.sh -ot .test-image && exit 0
+  time sudo ./build-lxd-image.sh
```

```suggestion
  time {{ if github_actions == "true" { "sudo" } else { "" } }} ./build-lxd-image.sh
```
only force sudo if on CI